common/djangoapps/third_party_auth/saml_configuration/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+    Viewset for auth/saml/v0/samlconfiguration/
```

Comment:
I just noticed this was wrong. 
I'm also just not sure if I should use `_` or not. `_` follows python style convention, however, given all of the other saml api endpoints are not using underscores, I feel like I should make it match (I'd ideally make it match by renaming all of those endpoints, but I don't really feel like there's value in that much modification at this point)

Reply:
hmm yeah using underscores seem desirable, but would be a major version change I guess if were to change the others...

Reply:
hmm pretty sure I did `auth/saml/v0/provider_config` and `auth/saml/v0/provider_data` for actual url patterns

Reply:
it does look like the urls are using underscores. I was just confused by the docstrings not using the underscores, ie: `auth/saml/v0/providerdata/` as the docstring for `saml_provider_data_router.register(r'provider_data', SAMLProviderDataViewSet, basename="saml_provider_data")`

Reply:
ah got it, yeah I think that could have ben done better indeed. no worries for this pr though